src/moveit_visual_tools.cpp

```diff
@@ -1517,22 +1517,14 @@ void MoveItVisualTools::publishRobotState(const moveit_msgs::DisplayRobotState&
   ros::spinOnce();
 }
 
-bool MoveItVisualTools::hideRobot()
+void MoveItVisualTools::hideRobot()
```

Comment:
i don't see a reason to remove the bool and change API, please restore

Reply:
+1. Please, also restore the corresponding return value.